python_modules/dagster/dagster/_core/definitions/declarative_scheduling/serialized_objects.py

```diff
@@ -31,6 +30,10 @@
 
 T = TypeVar("T")
 
+# state objects that store arbitrary data between ticks
+# if a need for new types arises, they can be added here, as long as they are serializable
+SerializableState = Union[None, AssetSubset, Sequence[ValidAssetSubset]]
```

Comment:
I don't think we should assign a name to this union. It's too general of name given what it is.

Can we just canonicalize to 'Optional[Sequence[ValidAssetSubset]]`?